agent/engine/docker_client_test.go

```diff
@@ -1282,3 +1282,67 @@ func TestMetadataFromContainerHealthCheckWithNoLogs(t *testing.T) {
 	metadata := metadataFromContainer(dockerContainer)
 	assert.Equal(t, api.ContainerUnhealthy, metadata.Health.Status)
 }
+
+func TestCreateVolumeTimeout(t *testing.T) {
+	timeout := 1*time.Second
```

Comment:
is the test actually waiting for this duration? If yes, can this be reduced to something in the order of milliseconds (Example: `10 ms`)?